compiler/one-cmds/one-import-bcq

```diff
@@ -184,30 +184,41 @@ show_err_onexit()
 
 trap show_err_onexit ERR
 
-# generate temporary preserved pb file
-echo "${DRIVER_PATH}/preserve_bcq_info" --input_path ${INPUT_PATH} \
---output_path "${TMPDIR}/${MODEL_NAME}_preserved.pb"  > "${OUTPUT_PATH}.log"
+# Generate BCQ information metadata
+# If model has no BCQ information or invalid information, pb file is not changed.
+echo "${DRIVER_PATH}/generate_bcq_metadata" \
```

Comment:
If you look at `one-import-tf`, `CONVERT_SCRIPT` is used to for the command and used for `echo` and execution.
Can you revise these lines like that?

Reply:
As another modification is needed, I will apply at this PR :)